Gemfile

```diff
@@ -66,7 +67,7 @@ gem 'airbrake'
 
 gem 'company_register', github: 'internetee/company_register', branch: :master
 gem 'e_invoice', github: 'internetee/e_invoice', branch: :master
-gem 'lhv', github: 'internetee/lhv', branch: :master
+gem 'lhv', github: 'internetee/lhv', branch: 'remove-keystores-gem'
```

Comment:
Suppose we should add the PR from 'remove-keystores-gem' to lvh/master to this PR dependencies, and we'll need to renew lock-file as well.

Reply:
Correct, you're right. At the moment this branch has PR #1646 changes applied so I was able to continue with development.